packages/@aws-cdk/aws-cloud9/package-lock.json

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
We don't use package-lock.json files now that we moved to Yarn. Feel free to remove this file.